app/src/main/java/org/asteroidos/sync/connectivity/IConnectivityService.java

```diff
@@ -0,0 +1,39 @@
+package org.asteroidos.sync.connectivity;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+/**
+ * A connectivity service is a module that can exchange data with the watch. It has to implement {@link IService}
+ * and additional functions regarding connectivity from {@link IConnectivityService}.
+ */
+public interface IConnectivityService extends IService {
+
+    enum Direction{
```

Comment:
Does it really make sense though? Can't certain characteristics support both write and notify?